internal/geometry.go

```diff
@@ -0,0 +1,12 @@
+package internal
+
+// PixelSize describe an object width and height in Pixels
+type PixelSize struct {
+	Width  int // The number of pixels along the X axis.
```

Comment:
As per slack discussions, if we really need this to differentiate then the Width should probably be WidthPx so that it's clear when autocompling what the type is rather than just in the field type or parameter lists.

Reply:
Ok, will do.